### ISSUE TITLE
Comment Question parsing

### ISSUE THREAD

**Issue by nickjillings** [BrechtDeMan/WebAudioEvaluationTool#206]
No passing yet for the on-page custom comment-question nodes.

A file structure may follow something like:
```
saves
|---page
        |---comment1.csv
        |---comment2.csv
        |---comment3.csv
```

Each csv should have the file-save name followed by the result type for each node. This will allow for multi-entry data types (such as the checkbox and radio) to be contained in the same scope as the single entry (question, number or slider).

**Comment by BrechtDeMan** [comment]
Separate from #205?

**Comment by nickjillings** [comment]
Yeah #205 is for the survey but this is for the on-page comments. Maybe they can be wrapped into one...

**Comment by BrechtDeMan** [comment]
Maybe but not necessarily. I was genuinely wondering. Perhaps keeping them apart is simpler.

**Comment by nickjillings** [comment]
It probably is, that way people can say "I've done these ones" with distinct names instead of names not directly infering what they do: `survey_parser.py` and `commentquestion_parser.py` make sense. Maybe the comment-question should be combined with comments though

**Comment by BrechtDeMan** [comment]
> Maybe the comment-question should be combined with comments though

Yep this could work. Folder per page containing a file for each fragment comment, and a file for each commentquestion, one line per subject. Should be an incremental change to the commentparser.
